### ISSUE TITLE
Incompatible with nestjs v7

### ISSUE THREAD

**Issue by BenWildeman** [nestjs/terminus#606]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

When using nestjs v7, eslint will throw errors about not being able to find `microservice-configuration.interface`

## Expected behavior

to work with v7

## Environment

<pre><code>
Nest version: 7.0.5

 
For Tooling issues:
- Node version: All compatible
- Platform:  All

Others:

</code></pre>

**Comment by BrunnerLivio** [comment]
Duplicate of #596 and #594. 

7.0.0 compatibility is on the way, check out #593

**Comment by hazzo** [comment]
Hi got one question: installing the new terminus version and running the server displays the following errors:

```
node_modules/@nestjs/terminus/dist/health-indicator/database/typeorm.health.d.ts:1:33 - error TS2307: Cannot find module 'typeorm'.

1 import type { Connection } from 'typeorm';
                                  ~~~~~~~~~

node_modules/@nestjs/terminus/dist/health-indicator/microservice/grpc.health.d.ts:1:34 - error TS2307: Cannot find module '@nestjs/microservices'.

1 import type { GrpcOptions } from '@nestjs/microservices';
                                   ~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@nestjs/terminus/dist/health-indicator/microservice/microservice.health.d.ts:1:93 - error TS2307: Cannot find module '@nestjs/microservices'.

1 import { GrpcOptions, MqttOptions, NatsOptions, RedisOptions, RmqOptions, TcpOptions } from '@nestjs/microservices';
                                        
```

Terminus needs `@nestjs/microservices` and `@nestjs/typeorm`?? If so, they not seem to appear in the `package-lock.json` as required dependencies.

**Comment by BrunnerLivio** [comment]
@hazzo still need to fix that, at the moment you will need to add `”skipLibCheck”: true` in your tsconfig to bypass that

**Comment by hazzo** [comment]
Okey cool thanks. I solved meanwhile by adding the corresponding dependencies.

**Comment by BrunnerLivio** [comment]
Resolved in [7.0.0](https://github.com/nestjs/terminus/releases/tag/7.0.0) :tada: 

@hazzo also the `skipLibCheck` is now resolved
